src/sqlancer/common/query/ExpectedErrors.java

```diff
@@ -54,14 +56,35 @@ public static ExpectedErrors from(String... errors) {
         return expectedErrors;
     }
 
+    public static ExpectedErrors from(Pattern... errors) {
```

Comment:
Btw, I think we could also provide convenience classes that make adding regular expressions less tedious (e.g., `fromRegexes(String... errors)` that takes the strings and instantiates `Pattern` objects.